### ISSUE TITLE
NTR: Vernal conjunctivitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3943]
New term request: Vernal conjunctivitis. Definition: \
Source: \
Parent: \
Disease association:

**Comment by pnrobinson** [comment]
@psergouniotis
I think this is best characterized as a disease with these features:
- https://hpo.jax.org/app/browse/term/HP:0025350 (Giant conjunctival papillae)
- one of the terms from Conunctivitis (https://hpo.jax.org/app/browse/term/HP:0000509)

- [ ] Maybe the primary label for the HPO term Conjunctivitis should be Conjunctival injection?

http://eyewiki.aao.org/Vernal_Keratoconjunctivitis

Conjunctival signs include diffuse conjunctival injection and upper tarsal giant papillae. These are discrete >1mm in diameter that characteristically have flattened tops which sometimes demonstrate stain with fluorescein [1, 4]. Additionally, these giant papillae can sometimes be seen near the limbus, and while relatively uncommon symblepharon formation and conjunctival fibrosis can occur [5].

**Comment by pnrobinson** [comment]
updated definition.

**Comment by psergouniotis** [comment]
So will vernal conjunctivitis be added in HPO (as a subclass of "conjunctivitis")? 
(perhaps I agree it is more a disease than a phenotype)

I did think about merging "Conjunctivitis" (HP:0000509) with "Conjunctival hyperemia" (HP:0030953). They are not synonyms but many people would use the one and mean the other. The issue with "Conjunctivitis" (and its subclasses) is that it is perhaps more a disease than a phenotype.

I do not have any strong feelings so, if you want, consider suppressing "conjunctivitis"

**Comment by pnrobinson** [comment]
If we agree that vernal conjunctivitis is a disease then it should not be an HPO term.
I do not think we can entirely suppress Conjunctivitis since it is a sign of so many other diseases. However, maybe we should add something to the comments of the term Conjunctival hyperemia to indicate this.

**Comment by pnrobinson** [comment]
Conjunctival injection is already a synonym of Conjunctival hyperemia. I do not think that Conjunctivitis is an exact synonym. I think the best thing is to leave things as they are for now.
